scripts/SANS/sans/gui_logic/presenter/beam_centre_presenter.py

```diff
@@ -128,6 +130,12 @@ def copy_centre_positions(self, state_model):
         self._beam_centre_model.front_pos_1 = getattr(state_model, "front_pos_1")
         self._beam_centre_model.front_pos_2 = getattr(state_model, "front_pos_2")
 
+    def set_scale_to_meters(self, enable: bool) -> None:
```

Comment:
Very minor suggestion, but would it be clearer to rename this to `set_scale`, just to make it a bit clearer from the name that it's not always going to set the scale to meters? The boolean parameter could then potentially be re-named to `is_meters`.